reactNativeProject/src/screens/Main/components/TodoList/index.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import TodoList from "./layout";
+import PropTypes from "prop-types";
+
+import { actionCreators } from "../../../../redux/item/actions";
+
+const mapDispatchToProps = dispatch => ({
+  removeItem: index => {
+    dispatch(actionCreators.removeItem(index));
+  },
+
+  toggleItemCompleted: index =>
+    dispatch(actionCreators.toggleItemCompleted(index))
+});
+
+const mapStateToProps = store => ({
+  items: store.item.items
+});
+class TodoListContainer extends Component {
```

Comment:
You don´t actually need this class. You can `connect` the layout to the store, and have the props you defined in `mapDispatchToProps` and `mapStateToProps` go directly to the layout.

You would do something like 
```
export default connect(
  mapStateToProps,
  mapDispatchToProps
)(TodoList);
```
and rename the objects in `mapDispatchToProps` to match the props your layout component is expecting. `removeItem` would be renamed `onRemoveItem` and `toggleItemCompleted` to `onToggleItemCompleted`